### PR TITLE
Fix grammar in DG

### PR DESCRIPTION
**Pull request by yixiann**

### CODE REVIEW

File: docs/_dg/HowToUseDeveloperGuide.md [hunk 1]
```diff
@@ -14,13 +14,13 @@ The Developer Guide has six main components:
 1. [Instructions for manual testing](#instructions-for-manual-testing)
 1. [Effort](#effort)
 
-As a **developer**, the first three sections would be most applicable to you. The section on [Design](#design) gives you a high-level overview of how the application is structured and what the key components are in FoodRem. After reading the section on design, the section on [Implementation](#implementation) covers how items, tags, statistics and general features are implemented. Lastly, the section on [Documentation, Configuration and Other Guides](#documentation-configuration-and-other-guides)
+As a **developer**, the first three sections would be most applicable to you. The section on [Design](#design) gives you a high-level overview of how the application is structured and what the key components of FoodRem. After reading the section on design, the section on [Implementation](#implementation) covers how items, tags, statistics, and general features are implemented. Lastly, the section on [Documentation, Configuration, and Other Guides](#documentation-configuration-and-other-guides)
```
Comment (RichDom2185) [inline docs/_dg/HowToUseDeveloperGuide.md:17]:
```suggestion
As a **developer**, the first three sections would be most applicable to you. The section on [Design](#design) gives you a high-level overview of how the application is structured and what the key components of FoodRem are. After reading the section on design, the section on [Implementation](#implementation) covers how items, tags, statistics, and general features are implemented. Lastly, the section on [Documentation, Configuration, and Other Guides](#documentation-configuration-and-other-guides)
```

File: docs/_dg/NFRs.md [hunk 2]
```diff
@@ -1,7 +1,7 @@
 <!-- markdownlint-disable-file first-line-h1 -->
 1. The product should work on any mainstream OS (Windows, macOS, Linux) that has Java 11 or above installed.
-1. The product should be able to hold up to 1000 items without a noticeable sluggishness in performance for typical usage.
-1. A user with above average typing speed for regular English text (i.e. not code, not system admin commands) should be able to accomplish most of the tasks faster using commands than using the mouse.
+1. The product should be able to hold up to 1000 items without noticeable sluggishness in performance for typical usage.
+1. A user with above average typing speed for regular English text (i.e. not code or system admin commands) should be able to accomplish most of the tasks faster using commands than using the mouse.
```
Comment (RichDom2185) [inline docs/_dg/NFRs.md:4]: Phrasing could be better I think.

```suggestion
1. A user with above average typing speed for regular, English text (not code, nor system admin commands) should be able to accomplish most of the tasks faster using commands than using the mouse.
```

**Review by RichDom2185 — APPROVED** [review]
LGTM, thanks! Just some comments below:

File: docs/_dg/NFRs.md (hunk 2 above)
Comment (yixiann) [reply docs/_dg/NFRs.md]: Okay

File: docs/_dg/HowToUseDeveloperGuide.md (hunk 1 above)
Comment (yixiann) [reply docs/_dg/HowToUseDeveloperGuide.md]: Thanks for the catch

**Review by RichDom2185 — APPROVED** [review]
LGTM, thanks!

**Pull request merged**